README.md

```diff
@@ -1,8 +1,3 @@
-# Pytket Extensions
-
-This repository contains the pytket-stim extension, using Quantinuum's
-[pytket](https://cqcl.github.io/tket/pytket/api/index.html) quantum SDK.
-
 # pytket-stim
 
 [Pytket](https://cqcl.github.io/tket/pytket/api/index.html) is a python module for interfacing
```

Comment:
Didn't you say you wanted to get rid of this line from the README as well?

If you keep this sentance I'd suggest using "optimising" instead of "optimisation"

Reply:
I have updated that now